widgy/contrib/widgy_mezzanine/models.py

```diff
@@ -62,6 +69,39 @@ class PageSelectRelatedManager(SelectRelatedManager, PageManager):
     """
     use_for_related_fields = True
 
+    def get_widgy_owners(self, item):
+        user = current_request().user
```

Comment:
Why are you doing this? I'd expect any use of a hack like this to have a darn good explanation.